packages/web-components/src/message-bar/README.md

```diff
@@ -15,37 +15,33 @@ The Fabric WC3 MessageBar extends `FASTElement`
 ### Template
 
 ```html
-<template
-  role="status"
-  layout="${x => x.layout}"
-  shape="${x => x.shape}"
-  intent="${x => x.intent}"
-  aria-live="${x => x.politeness}"
-  aria-labelledby="${x => x.ariaLabelledby}"
->
-  <slot></slot>
-  <slot name="actions"></slot>
-  <slot name="close"></slot>
+<template>
```

Comment:
You can remove the unused template here since you are no longer binding anything to it

Reply:
Nice catch.